src/rosdep2/main.py

```diff
@@ -296,6 +296,10 @@ def _rosdep_main(args):
                            "locally ignore a rosdep key is creating a local rosdep rule "
                            "with an empty list of packages (include it in "
                            "/etc/ros/rosdep/sources.list.d/ before the defaults).")
+    parser.add_option("--install-method",
```

Comment:
I think this needs a verb, a good way to tell is to read out loud the finial command name, "rodep db --install-method", doesn't really say what `db` is doing with the provided install method. What about `--filter-for-install-method INSTALL_METHOD`? Also should it be `install-method` or `installer`?